litmus-portal/frontend/src/components/Unimodal/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Modal from '@material-ui/core/Modal';
+import Button from '@material-ui/core/Button';
+import useStyles from './styles';
+
+/* DelUser, NewUserModal, ResetModal need to be shifted */
+
+interface UnimodalProps {
+  children?: any;
+  isOpen: boolean;
+  handleClose: any;
```

Comment:
handleClose would be a function not `any` you can take a reference from `InputField` component